src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -239,6 +239,14 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
 
 # Local Gateway
 register_cli_argument('network local-gateway', 'local_network_gateway_name', name_arg_type, completer=get_resource_name_completion_list('Microsoft.Network/localNetworkGateways'), id_part='name')
+register_cli_argument('network local-gateway', 'asn', arg_group='BGP Peering', help='Autonomous System Number to use for the BGP settings.')
+register_cli_argument('network local-gateway', 'bgp_peering_address', arg_group='BGP Peering', help='IP address from the OnPremise VPN\'s subnet to use for BGP peering.')
+register_cli_argument('network local-gateway', 'peer_weight', arg_group='BGP Peering', help='Weight (0-100) added to routes learned through BGP peering.')
+register_cli_argument('network local-gateway', 'local_address_prefix', nargs='+', help='List of CIDR block prefixes representing the address space of the OnPremise VPN\'s subnet.')
```

Comment:
This should be plural `local_address_prefixes`. I know SDK also uses singular, but to me it is bug